tests/jballerina-unit-test/src/test/resources/test-src/services/client_resource_return_negative.bal

```diff
@@ -0,0 +1,103 @@
+type CustomType1 float|int[]|client object{}|int|string;
```

Comment:
Let's add the Copyright header

Reply:
Updated 2504be25de6173179816f9cda84efa8d7ab3883f